ports/stm32/Makefile

```diff
@@ -23,12 +25,12 @@ FROZEN_MPY_DIR ?= modules
 include $(TOP)/py/py.mk
 
 LD_DIR=boards
-CMSIS_DIR=$(TOP)/lib/stm32lib/CMSIS/STM32$(MCU_SERIES_UPPER)xx/Include
 MCU_SERIES_UPPER = $(shell echo $(MCU_SERIES) | tr '[:lower:]' '[:upper:]')
+CMSIS_DIR=lib/stm32lib/CMSIS/STM32$(MCU_SERIES_UPPER)xx/Include
```

Comment:
Why this change?  Note that `CMSIS_MCU_HDR` also depends on `CMSIS_DIR` and would need to be updated too.

Reply:
Yeah I've already removed this, mistakenly thought I needed this with an earlier superproject structure.